akka-typed/src/main/scala/akka/typed/patterns/Restarter.scala

```diff
@@ -20,7 +20,7 @@ final case class Restarter[T, Thr <: Throwable: ClassTag](initialBehavior: Behav
   behavior: Behavior[T] = initialBehavior) extends Behavior[T] {
 
   private def restart(ctx: ActorContext[T]): Behavior[T] = {
-    try behavior.management(ctx, PreRestart) catch { case NonFatal(_) ⇒ }
+    try behavior.management(ctx, PreRestart) catch { case NonFatal(_) ⇒ } // TODO shouldn't it stop?
```

Comment:
I was think of the classic behavior to stop in case of exception from constructor or `preStart`, I guess it depends on the new behavior is same or not